marketplaceItems/appreciate/appResources/appData/resources/appreciate_ui.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<!--
+    Appreciate
+    Created by Zach Fox on 2019-01-30
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<html>
+    <head>
+        <title>Appreciate</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+        <!--I modified jscolor to prevent it from printing the color code to the value of the input onchange. -->
+        <script type="text/javascript" src="js/jscolor.js"></script>
+    </head>
+
+    <body>
+        <div id="loadingContainer">
+        </div>
+
+        <div id="mainContainer">
+            <div id="titleBarContainer">
+                <div id="titleText" style="font-weight:bold;">
```

Comment:
Not a fan of styles in the html, but to each his own :)

Reply:
inline styles have a great deal of usefulness! especially when this operation would take three lines of CSS to accomplish the same thing elsewhere.

Reply:
👍